.github/workflows/check-docs.yml

```
@@ -0,0 +1,14 @@
+﻿name: Check the Docfx site
+
+on:
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)